docs/spark/how-to-guides/hdinsight-deploy-methods.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Submit a .NET for Apache Spark job to Azure HDInsight
+description: Learn how to submit a .NET for Apache Spark job to Azure HDInsight using spark-submit and Apache Livy.
+ms.date: 11/14/2019
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# Submit a .NET for Apache Spark job to Azure HDInsight
+
+There are two ways to deploy your .NET for Apache Spark job to HDInsight: `spark-submit` and Apache Livy.
```

Comment:
Should we add a note/link here asking users to view the HDI doc before this one?

Reply:
I don't think it's necessary as a prereq, but it is linked in "next steps" as additional reading.

Reply:
Okay makes sense!